public/docs/ts/latest/tutorial/toh-pt1.jade

```diff
@@ -32,164 +33,143 @@ include ../_quickstart_repo
-  package in our Angular module. We add it to the `NgModule` decorator's `imports` array. This array contains the list
-  of external modules used by our application.
-  Now we have included the forms package which includes `ngModel`.
+  Before using two-way data binding for **form inputs**, import the `FormsModule`
+  package in your Angular module. Add the `FormsModule` to the `NgModule` decorator's `imports` array, which contains the list
+  of external modules that the app uses.
+  Now you have included the forms package that includes `ngModel`.
 
 +makeExample('toh-1/ts/app/app.module.ts', '', 'app.module.ts (FormsModule import)')
 
 .l-sub-section
   :marked
     Learn more about the `FormsModule` and `ngModel` in the
     [Forms](../guide/forms.html#ngModel) and
-    [Template Syntax](../guide/template-syntax.html#ngModel) chapters.
+    [Template Syntax](../guide/template-syntax.html#ngModel) pages.
 
 :marked
-  Let’s update the template to use the  **`ngModel`** built-in directive for two-way binding.
-
-  Replace the `<input>` with the following HTML
+  Now update the template to use the  **`ngModel`** built-in directive for two-way binding.
+  In app.component.ts, replace the `<input>` with the following HTML:
 
 code-example(language="html").
   &lt;input [(ngModel)]="hero.name" placeholder="name">
 
 :marked
-  The browser refreshes. We see our hero again. We can edit the hero’s name and
+  The browser refreshes. Now you can edit the hero's name and
   see the changes reflected immediately in the `<h2>`.
 
 .l-main-section
 :marked
-  ## The Road We’ve Travelled
-  Let’s take stock of what we’ve built.
-
-  * Our Tour of Heroes uses the double curly braces of interpolation (a kind of one-way data binding)
-  to display the application title and properties of a `Hero` object.
-  * We wrote a multi-line template using ES2015’s template strings to make our template readable.
-  * We can both display and change the hero’s name after adding a two-way data binding to the `<input>` element
-  using the built-in `ngModel` directive.
-  * The `ngModel` directive also propagates changes to every other binding of the `hero.name`.
-
-  Run the <live-example></live-example> for this part.
-
+  ## The road you've travelled
+  Let's take stock of what you've built.
+
+  * The Tour of Heroes app uses the double curly braces of interpolation (a type of one-way data binding)
+  to display the app title and properties of a `Hero` object.
+  * You wrote a multi-line template using ES2015's template strings to make the template readable.
+  * You added a two-way data binding to the `<input>` element
+  using the built-in `ngModel` directive. This binding both displays the hero's name and allows users to change it.
+  * The `ngModel` directive propagates changes to every other binding of the `hero.name`.
+
+  **Try it out**. Here's a link to a <live-example></live-example>.
```

Comment:
I prefer the way this sentence was since we don't need to specify that it's a link. That should be clear by the styles. We don't want it to read like "click here for more".